test/TestConfig/scripts/testKitGen/resultsSummary/resultsSum.pl

```diff
@@ -186,6 +194,20 @@ sub resultReporter {
 
 	print "++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++\n";
 
+	if ($numOfFailed != 0) {
+		my $buildParam =  "";
+		if ($buildList ne '') {
+			$buildParam = "&BUILD_LIST=" . $buildList;
+		}
+		print "To rebuild the failed test in a jenkins job, copy the following link and fill out the <Jenkins URL> and <FAILED test target>:\n";
+		print "<Jenkins URL>/parambuild/?PERSONAL_BUILD=false&JDK_VERSION=$jdkVersion&JDK_IMPL=$jdkImpl$buildParam&TARGET=<FAILED test target>\n\n";
```

Comment:
Please remove `PERSONAL_BUILD=false`. This is no longer needed.

Reply:
removed